src/GraphQL/AstExtensions.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using GraphQL.Language.AST;
+
+namespace GraphQL
+{
+    /// <summary>
+    /// Provides extension methods for AST nodes.
+    /// </summary>
+    public static class AstExtensions
+    {
+        /// <summary>
+        /// Returns the original string that was parsed into the specified node.
+        /// </summary>
+        public static string ToString(this INode node, Document document)
```

Comment:
```suggestion
        public static string TakeFrom(this INode node, Document document)
```

It seems to me that the special name here instead of `ToString()` will be more appropriate.

Reply:
But I'm not sure.